ocaml/idl/datamodel_host.ml

```diff
@@ -1325,6 +1325,34 @@ let host_query_ha = call ~flags:[`Session]
     ~allowed_roles:_R_LOCAL_ROOT_ONLY
     ()
 
+  let host_sched_gran = Enum("host_sched_gran", [
+    "core", "core scheduling";
+    "cpu", "CPU scheduling";
+    "socket", "socket scheduling";
+  ])
+
+  let set_sched_gran = call
+    ~name:"set_sched_gran"
+    ~lifecycle:[Prototyped, rel_next, ""]
+    ~doc:"Sets xen's sched-gran on a host. See: https://xenbits.xen.org/docs/unstable/misc/xen-command-line.html#sched-gran-x86"
+    ~params:[
+      Ref _host, "host", "The host";
+      host_sched_gran, "sched", "The sched-gran to apply to a host"
```

Comment:
The usual name for the value parameter of setters in the API is `value`.

Reply:
The usual name for the first parameter is actually `self` – for the autogenerated getters and setters at least. In this case, I don't mind `host`, because we have a lot of precedents.